docs/user_guide.md

```diff
@@ -556,76 +531,184 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
+1. Click **Add** to connect Harbor to the scanner.
+1. Optionally repeat the procedure to add more scanners.
+1. If you configure multiple scanners, select one and click **Set as Default** to designate it as the default scanner.
+
+### Updating the Vulnerability Metadata in the Database
+
+Vulnerability scanners depend on the vulnerability metadata to complete the analysis process. After the first initial installation, the vulnerability scanner automatically starts to update the metadata database from different vulnerability repositories. The database update might take a while, based on the data size and network connection. 
+
+Once the database is ready, the timestamp of the last update is shown in the **Interrogation Services** > **Vulnerability** tab. 
 ![browse project](img/clair_ready.png)
 
-**Scanning an image** 
+Until the database has been fully populated, the timestamp is replaced by a warning symbol. When the database is ready, you can scan images individually or scan all images across all projects.
+
+### Scan Individual Images
+
+1. Log in to the Harbor interface with an account that has at least project administrator privileges.
+1. Go to **Projects** and select a project. 
+1. Select the **Scanner** tab.
+
+   The **Scanner** tab shows the details of the scanner that is currently set as the scanner to use for this project.
+   
+   ![Project scanner tab](img/project-scanners.png)
+1. Click **Edit** to select a different scanner from the list of scanners that are connected to this Harbor instance, and click **OK**.
+
+   ![Project scanner tab](img/select-scanner.png)
+1. Select the **Repositories** tab and select a repository.
+
+   For each tag in the repository, the **Vulnerabilities** column displays the vulnerability scanning status and related information.
+   
+   ![Tag vulnerability status](img/tag-vulnerability-status.png)
+1. Select a tag, or use the check box at the top to select all tags in the repository, and click the **Scan** button to run the vulnerability scan on this image.
+
+   ![Scan an image](img/scan_image.png)
+
+   **NOTE**: You can start a scan at any time, unless the status is **Queued** or **Scanning**. If the database has not been fully populated, there is a warning message at the footer of the **Projects** > **Repositories** view and you cannot run the scan. The following statuses are displayed in the **Vulnerabilities** column:
+   
+   * **Not Scanned:** The tag has never been scanned.
+   * **Queued:** The scanning task is scheduled but has not run yet.
+   * **Scanning:** The scanning task is in progress and a progress bar is displayed.
+   * **View log:** The scanning task failed to complete. Click **View Log** link to view the related logs.
+   * **Complete:** The scanning task completed successfully.
+
+   If the process completes successfully, the result indicates the number of vulnerabilities with each severity level and the number of vulnerabilities that are fixable.
```

Comment:
This is the overall vulnerability summary:


The icon indicates the overall severity of such image
- Critical if the number of critical vulnerabilities > 0
- High if the number of high vulnerabilities > 0
- ...

The total number of found vulnerabilities with kinds of severities
The number of fixable vulnerabilities

Reply:
@steven-zou I reworded as follows:

 If the process completes successfully, the result indicates the total number of vulnerabilities found for each severity level, and the number of fixable vulnerabilities.

   * **Red:** At least one critical vulnerability found
   * **Orange:** At least one high level vulnerability found
   * **Yellow:** At least one medium level vulnerability found
   * **Blue:** At least one low level vulnerability found
   * **Green:** No vulnerabilities found
   * **Grey:** Unknown vulnerabilities

Changes in https://github.com/goharbor/harbor/pull/10029/commits/a1eac99a49aa192cb5048c5eaa8a4e2a756cd134.

Reply:
>the result indicates the total number of vulnerabilities found for each severity level, and the number of fixable vulnerabilities.

-> `the result indicates the overall severity with the total number of vulnerabilities found for each severity level, and the number of fixable vulnerabilities`?

Reply:
I think Stuart's is a little better :)